vendor/github.com/Microsoft/hcsshim/.golangci.yml

```diff
@@ -107,4 +107,4 @@ issues:
     - path: internal\\hcserror\\
       linters:
         - stylecheck
-      Text: "ST1003:"
\ No newline at end of file
+      Text: "ST1003:"
```

Comment:
Please do not modify vendored files

Reply:
Sure @AkihiroSuda. I am taking care of that.